src/main/java/com/fundynamic/d2tm/Game.java

```diff
@@ -37,6 +38,11 @@ public void initStatesList(GameContainer container) throws SlickException {
         PlayingState playingState = new PlayingState(
                 container,
                 terrainFactory,
+                new Shroud(
+                    new Image("shroud_edges.png"),
+                    TILE_WIDTH,
```

Comment:
i took care of these `TILE_WIDTH` and `TILE_HEIGHT` stuff elsewhere (in the nuclear missile branch). Merging will hurt, but no probs. Its ugly having to pass it everywhere isnt it?